src/olm/session/message_key.rs

```diff
@@ -43,22 +43,43 @@ impl Drop for RemoteMessageKey {
 }
 
 impl MessageKey {
+    #[cfg(feature = "low-level-api")]
```

Comment:
Do we need to change the constructor? Can't we:

1. Make the `MessageKey` pub
2. Only re-export it publicly if the feature flag is set.